sv_pipeline/genome/load_data.py

```diff
@@ -38,7 +38,8 @@
                                              hl.missing(hl.tstr)),
     'gnomad_svs_AF': lambda rows: rows.info.gnomAD_V2_AF,
     'pos': lambda rows: rows.locus.position,
-    'filters': lambda rows: hl.array(rows.filters.filter(lambda x: x != 'PASS')),
+    'filters': lambda rows: hl.array(rows.filters.filter(lambda x: (x != 'PASS') & (x != 'BOTHSIDES_SUPPORT'))),
```

Comment:
since you use the string `'BOTHSIDES_SUPPORT'` twice it is better to make a constant